src/audio/aria/aria.toml

```diff
@@ -0,0 +1,25 @@
+	// # Aria module config
```

Comment:
heh, looks like GH is letting us know toml has a comment style.

Reply:
```suggestion
	// Aria module config
```
Don't give the impression that two "levels" of comments are possible. The C pre-processor either leaves comments in the source or it deletes them; it's not capable (or at least not trivial) to pass comments to the generated .toml file.

Reply:
Ping?

Reply:
this and the double brackets, looks like this is invalid

Reply:
> this and the double brackets, looks like this is invalid

@cujomalainey the comment is just removed by the preprocessor. Double brackets are valid, they're used in our present sources

Reply:
> Ping?

@marc-hb sure, we can remove hashes there, no strong opinion here

Reply:
> Don't give the impression that two "levels" of comments are possible. The C pre-processor either leaves comments in the source or it deletes them; it's not capable (or at least not trivial) to pass comments to the generated .toml file.

I found a workaround:
- https://github.com/zephyrproject-rtos/zephyr/pull/67019

Demo in https://github.com/marc-hb/sof/commits/toml-REM/